ovos_bus_client/session.py

```diff
@@ -443,8 +443,13 @@ def from_message(message: Message = None):
                 sess["lang"] = lang
             sess = Session.deserialize(sess)
         else:
+            if message:
+                LOG.warning(f"No session context in message:{message.msg_type}")
+                LOG.debug(f"Add `session: SessionManager.default_session` to "
+                          f"message.context. context={message.context}")
```

Comment:
this isnt quite right, it should instead say to update ovos-bus-client on the message emitting code to ensure a session is present, default_session is not always the correct thing to do, and should not need to be done by devs manually. the correct session is assigned on bus client init method

Reply:
Updated the log message. My assumption here is that this will come from code interfacing with the messagebus directly, without `ovos-bus-client` so I do want to include some note about what context is missing (i.e. some JS client interfacing with the messagebus locally).